### PR TITLE
Add new condensed strokes

### PR DESCRIPTION
**Pull request by paulfioravanti**
This PR proposes to add a set of found condensed strokes not currently in Plover's `main.json`.

Notes:

- `"KW-GS": "{~|\"^}"` and `"KR-GS": "{^~|\"}"` work as expected as condensed strokes, but even though they're in the `punctuation-powerups.json` dictionary, I figured I'd not remove them from there, since perhaps there may be people that would use that dictionary, but not the condensed strokes one (...?).

### CODE REVIEW

File: dictionaries/condensed-strokes.json [hunk 1]
```diff
@@ -22,6 +22,7 @@
 "-B/HO*ELD": "behold",
 "-B/HO*LD": "behold",
 "-F/TP-PL": "of.",
+"-GT": "thing",
```
Comment (didoesdigital) [inline dictionaries/condensed-strokes.json:25]: @paulfioravanti what's the particular motivation for adding this to the condensed strokes dictionary? While it is indeed a condensed stroke that already works with existing Plover dictionary, it doesn't appear to follow any existing convention for phrasing briefs ending in GT and " thing". As it is shorter than the current preferred phonetic entry in the top X words dictionaries, Typey Type would prioritise the shorter entry, but I think the phonetic entry might be better to teach new steno students.

**Comment by didoesdigital** [comment]
Thanks @paulfioravanti ! A lot of useful entries to improve steno lookups in this PR 👏

File: dictionaries/condensed-strokes.json (hunk 1 above)
Comment (paulfioravanti) [reply dictionaries/condensed-strokes.json:25]: > condensed stroke that already works with existing Plover dictionary
Yep, that was pretty much the only reason I included it: I inadvertently stumbled on it, and there was no named entry for it in Plover.

> I think the phonetic entry might be better to teach new steno students
I can whole-heartedly agree with this, and wouldn't want to have what could be considered an unintuitive entry be prioritised by Typey Type. Will remove it :+1:

**Comment by didoesdigital** [comment]
In production! Thanks @paulfioravanti 🙏

**Pull request merged**